src/AccountServices.js

```diff
@@ -0,0 +1,147 @@
+var RemoteRoom = require("matrix-appservice-bridge").RemoteRoom;
+var RemoteUser = require("matrix-appservice-bridge").RemoteUser;
+var Twitter    = require('twitter');
+var log        = require('npmlog');
+var OAuth      = require('oauth');
+var TwitterHandler = require('./TwitterHandler.js').TwitterHandler;
+
+
+
+/**
+ * Construct a AccountServices.
+ * This class is a handler for conversation between users and the bridge bot to
+ * link accounts together
+ *
+ * @class
+ * @extends {external:TwitterHandler}
+ *
+ * @param  {matrix-appservice-bridge.Bridge}   bridge
+ * @param  app_auth OAuth authentication information
+ * @param  app_auth.consumer_key Twitter consumer key
+ * @param  app_auth.consumer_secret Twitter consumer secret
+ * @param  {TwitterDB}       storage
+ * @param  {MatrixTwitter}   twitter
+ */
+var AccountServices = function (bridge, app_auth, storage, twitter) {
+  TwitterHandler.call(this,bridge,null,"service");
+  this._app_auth = app_auth;
+  this._storage = storage;
+  this._twitter = twitter;
+  this._oauth = new OAuth.OAuth(
+    'https://api.twitter.com/oauth/request_token',
+    'https://api.twitter.com/oauth/access_token',
+    this._app_auth.consumer_key,
+    this._app_auth.consumer_secret,
+    '1.0A',
+    "oob",
+    'HMAC-SHA1'
+  );
+}
+
+AccountServices.prototype.processInvite = function (event, request, context){
+  var intent = this._bridge.getIntent();
```

Comment:
In general, you should [prefer composition over inheritance](https://en.wikipedia.org/wiki/Composition_over_inheritance). There is rarely a good reason to use inheritance in JS proper, and the lack of static typing makes base functions/variables like this surprising to find. Where was this sucker defined? CTRL+F in this file and seemingly nowhere! YAY! My next resolution step would be to check the constructor, see you're calling `TwitterHandler` then check in that file to see if it is there. It is, so YAY! All good, right? Right? Well yes, sorta, but you're now tethered to the *private* variables defined in *another file*. That's one of the problems with inheritance: children end up knowing far more about their parents than they really should do!